pcsx2/CDVD/CsoFileReader.cpp

```diff
@@ -0,0 +1,277 @@
+/*  PCSX2 - PS2 Emulator for PCs
+*  Copyright (C) 2002-2014  PCSX2 Dev Team
+*
+*  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+*  of the GNU Lesser General Public License as published by the Free Software Found-
+*  ation, either version 3 of the License, or (at your option) any later version.
+*
+*  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+*  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+*  PURPOSE.  See the GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License along with PCSX2.
+*  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "PrecompiledHeader.h"
+#include "AsyncFileReader.h"
+#include "CompressedFileReaderUtils.h"
+#include "CsoFileReader.h"
+#include "Pcsx2Types.h"
+#include "zlib_indexed.h"
```

Comment:
For PX_fseeko.  I see FileStream, but it doesn't seem to be unicode-safe.

Anyway, I've just moved the define into `CompressedFileReaderUtils.h` and included zlib directly.

-[Unknown]